src/network/message.rs

```diff
@@ -137,9 +146,12 @@ impl<N: Network, E: Environment> Message<N, E> {
             Self::PeerResponse(..) => 6,
             Self::Ping(..) => 7,
             Self::Pong(..) => 8,
-            Self::UnconfirmedBlock(..) => 9,
-            Self::UnconfirmedTransaction(..) => 10,
-            Self::Unused(..) => 11,
+            Self::PoolRegister(..) => 9,
+            Self::PoolRequest(..) => 10,
+            Self::PoolResponse(..) => 11,
+            Self::UnconfirmedBlock(..) => 12,
```

Comment:
note: this change must occur in tandem with https://github.com/AleoHQ/snarkOS/pull/1466 or a different version update

Reply:
The merge target should've been set to `feat/v12`

Reply:
it is, just double-checking :+1: